klsreader/protocol/controllerdata.py

```diff
@@ -40,7 +40,7 @@ def _unpackPacketA(self, data):
         (self.throttle, self.brakePedal, self.brakeSwitch, self.footSwitch,
         self.forwardSwitch, self.reverse, self.hallA, self.hallB, self.hallC,
         self.batteryVoltage, self.motorTemp, self.controllerTemp, self.settingDir,
-        self.actualDir) = unpack('2x5B4?3B2?3x', data)
+        self.actualDir) = unpack('!2x5B4?3B2?3x', data)
```

Comment:
Doesn't seem to break anything, and I don't see what future harm could result from it. I'll implement it on the next pull request.